src/test_series.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+FIB_TABLE = [(0, 0), (1, 1), (2, 1), (3, 2), (4, 3), (5, 5), (6, 8), (7, 13)]
+
+LUC_TABLE = [(0, 2), (1, 1), (2, 3), (3, 4), (4, 7), (5, 11), (6, 18), (7, 29)]
```

Comment:
These should contain one set of test inputs per line.  While your 'lines-of-code' will increase it will be much more readable.  This is not a bad things.